composer.json

```diff
@@ -53,11 +53,21 @@
         "post-autoload-dump": "Cake\\Composer\\Installer\\PluginInstaller::postAutoloadDump",
         "check": [
             "@test",
+            "@cs-setup",
             "@cs-check"
         ],
-        "cs-check": "phpcs --colors -p --standard=vendor/cakephp/cakephp-codesniffer/CakePHP ./src ./tests",
-        "cs-fix": "phpcbf --colors --standard=vendor/cakephp/cakephp-codesniffer/CakePHP ./src ./tests",
-        "test": "phpunit --colors=always"
+        "cs-check": " vendor/bin/phpcs --colors -n -p --extensions=php --ignore=/Migrations/,/Seeds/ ./config ./src ./tests ./plugins/*/*/config ./plugins/*/*/src ./plugins/*/*/tests",
+        "cs-fix": "vendor/bin/phpcbf --colors --extensions=php --ignore=/Migrations/,/Seeds/ ./config ./src ./tests ./plugins/*/*/config ./plugins/*/*/src ./plugins/*/*/tests",
+        "cs-setup": [
+            "vendor/bin/phpcs --config-set installed_paths vendor/cakephp/cakephp-codesniffer",
+            "vendor/bin/phpcs --config-set default_standard CakePHP",
+            "vendor/bin/phpcs --config-set colors 1"
+        ],
+        "test": "vendor/bin/phpunit --colors=always",
+        "update-dev": [
```

Comment:
Could we add `@cs-setup` to `post-install` instead of adding an additional setup command?

Reply:
Uhm... @fquffio  since `phpcs` is in `require-dev` what would happens if the `install` command was called with `--no-dev` option? Is there a way to exec `post-install` only in dev context?

Reply:
Anyway I think it was a good idea, maybe `post-update-cmd` would be the best place to have phpcs always well configured.
We could try to manage it in `BEdita\App\Console\Installer::postInstall`.
Maybe in another issue :smile:

Reply:
Initially I wanted to put `@cs-setup` in `post-updated-cmd` but, apart from the `--no-dev` problem mentioned before it's something you need only as a developer.

My proposal here for devs is to use `composer run-script update-dev` instead of `composer update`